### PR TITLE
[NEW] geoengine_project

### PR DESCRIPTION
**Pull request by JonathanNEMRY**
- Geolocalise your projects based on the related customer.

Note that this PR depends of this one: https://github.com/OCA/geospatial/pull/24

### CODE REVIEW

**Comment by nbessi** [comment]
You should avoid google map as defaut background. 
The google API is hard to integrate and problematic and was designed on purpose. 
We will probably depreacted it support soon.  It will not be integrated with openlayers 3

**Comment by nbessi** [comment]
LGTM iut will be nice if you can add a simple unit test that does a crud on the project

File: geoengine_project/README.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+===============================
+
+Geolocalise your projects based on the related customer.
+
+
+Installation
+============
+
+Take a look at the installation section in the description of the module 
+'base_geoengine'.
```
Comment (dreispt) [inline geoengine_project/README.rst]: I see `base_geoengine` nowhere

**Comment by max3903** [comment]
CLA fixed

**Comment by nbessi** [comment]
:+1:

File: geoengine_project/views/geo_project_view.xml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+<openerp>
+    <data>
+        <record id="project.open_view_project_all" model="ir.actions.act_window">
+            <field name="view_mode">kanban,tree,form,gantt,geoengine</field>
+        </record>
+
+        <record model="ir.ui.view" id="geo_project_project_view_form">
+            <field name="name">geo_project_project_form</field>
+            <field name="model">project.project</field>
+            <field name="type">form</field>
```
Comment (lmignon) [inline geoengine_project/views/geo_project_view.xml]: type is deprecated

File: geoengine_project/README.rst [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
```
Comment (lmignon) [inline geoengine_project/README.rst:15]: PLZ add a bug tracker section as defined in https://github.com/OCA/maintainer-tools/tree/master/template/module#bug-tracker

**Comment by lmignon** [comment]
@JonathanNEMRY I've rebased your changes on 8.0 and amended your commits to set you as author

**Comment by pedrobaeza** [comment]
The icon is good for the main module, but for this one it can be preferible to make a mix with the project icon and this one. This is a sample of this. Original one: https://github.com/odoomrp/odoomrp-wip/blob/8.0/quality_control/static/description/icon.png and derived one: https://github.com/odoomrp/odoomrp-wip/blob/8.0/quality_control_mrp/static/description/icon.png

**Comment by pedrobaeza** [comment]
It is also a bit large. A good size is 128x128

**Comment by JonathanNEMRY** [comment]
Hi @pedrobaeza
Thanks for your review. Maybe it could be a good idea to have a special logo depending on the subject of the module but for the moment all logos are the same into geospatial addons so I just used the same than the others. I propose to merge this one without changing the logo now and eventually discuss about it into another PR...what do you think @nbessi ?

File: geoengine_base_geolocalize/tests/test_geoengine_partner.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+import openerp.tests.common as common
+
+
+class TestGeoengineProjects(common.TransactionCase):
```
Comment (lmignon) [inline geoengine_base_geolocalize/tests/test_geoengine_partner.py]: The class name should be `TestGeoenginePartner` :smile:

**Comment by lmignon** [comment]
@JonathanNEMRY 2 issues remains: 
- Wrong classname for the test class in geoengine_project*
- The `Bug Tracker` section misses in the REAME.rst

**Comment by JonathanNEMRY** [comment]
Thanks @lmignon I will fix this

**Comment by pedrobaeza** [comment]
OK, @JonathanNEMRY, I will make a PR with the icons when this will be merged. Do you have the original SVG source?

**Comment by nbessi** [comment]
@JonathanNEMRY honnestly we can change the logo later at least there are all coherent within the repo. Also if we have a rule to change the logo I would like to integrate an OCA totem

**Comment by lmignon** [comment]
Ping @JonathanNEMRY

**Comment by JonathanNEMRY** [comment]
@nbessi Totaly agree with you: that was my point of view too

**Comment by lmignon** [comment]
:+1:  errors in Travis are unrelated to this pr

**Comment by JonathanNEMRY** [comment]
Is it ok to merge it now?

**Comment by lmignon** [comment]
@JonathanNEMRY Thank you for the 'git rebase'. Travis is now green. :+1:

**Comment by adrienpeiffer** [comment]
:+1: when travis is green

**Comment by JonathanNEMRY** [comment]
Seems that coverage has some difficulties...any ideas? (otherwise there are enough of ":+1:" I think :smile: )

**Comment by JonathanNEMRY** [comment]
Thanks @nbessi

**Pull request merged**